modules/configuring-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+//* Installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing-vsphere.adoc [UPI]
+//* installing-vsphere-network-customizations.adoc [UPI]
+//* installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: PROCEDURE
+[id="configuring-vsphere-regions-zones_{context}"]
+= Configuring regions and zones for a VMware vCenter
+You can modify the default installation configuration file, so that you can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. 
+
+[NOTE]
+====
+The default `install-config.yaml` file configuration from the previous release of {product-title} is deprecated. You can continue to use the deprecated default configuration, but the `openshift-installer` will prompt you with a warning message that indicates the use of deprecated fields in the configuration file. 
+====
+
+[IMPORTANT]
+====
+The example uses the `govc` command. The `govc` command is an open-source command available from VMware; it is not available from Red Hat. The Red Hat support team does not maintain the `govc` command. Instructions for downloading and installing `govc` are found on the VMware documentation website
+====
+
+.Prerequisites
+* You have an existing `install-config.yaml` installation configuration file.
+
+.Procedure
+
+. Use the `govc` command-line tool to create the `openshift-region` and `openshift-zone` vCenter tag categories.
++
+[IMPORTANT]
+====
+If you specify different names for the vCenter tag categories, the installation of the {product-title} cluster fails.  
+====
++
+[source,terminal]
+----
+$ govc tags.category.create -d "OpenShift region" openshift-region
+
+$ govc tags.category.create -d "OpenShift zone" openshift-zone
+----
+
+. Create a region tag for each region:
++
+[source,terminal]
+----
+$ govc tags.create -c <region_tag_category> <region_tag>
+----
+
+. Create a zone tag for each zone:
++
+[source,terminal]
+----
+$ govc tags.create -c <zone_tag_category> <zone_tag>
+----
+
+. Attach region tags to each vCenter datacenter object: 
++
+[source,terminal]
+----
+$ govc tags.attach -c <region_tag_category> <region_tag_1> /<datacenter_1>
+----
+
+. Attach the zone tags to each vCenter datacenter object:
++
+[source,terminal]
+----
+$ govc tags.attach -c <zone_tag_category> <zone_tag_1> /<datacenter_1>/host/vcs-mdcnc-workload-1
+----
+
+. Create the `install-config.yaml` file and configure the file to support operating multiple datacenters in a vSphere center.
```

Comment:
This is an unnecessary step. You already state that an existing `install-config.yaml` file is required. And because the installation doc is written in a step-wise fashion, users will have one.

Reply:
Removed and moved the note to the prereq step.